crates/rethnet_evm_napi/src/config.rs

```diff
@@ -198,9 +190,14 @@ impl TryFrom<ConfigOptions> for CfgEnv {
         cfg.chain_id = chain_id;
         cfg.spec_id = spec_id;
         cfg.limit_contract_code_size = limit_contract_code_size;
+        cfg.limit_initcode_size = limit_initcode_size;
         cfg.disable_block_gas_limit = disable_block_gas_limit;
         cfg.disable_eip3607 = disable_eip3607;
 
         Ok(cfg)
     }
 }
+
+fn bigint_to_maybe_usize(value: BigInt) -> Result<Option<usize>, napi::Error> {
```

Comment:
This function seems redundant now. You can just use `TryCast::<usize>::try_cast`, if I am not mistaken

Reply:
I think we need it, because we want a `Result<Option<usize>, _>`, but `TryCast::<usize>::try_cast` returns a `Result<usize, _>`

Reply:
Can't you just inline use `transpose` like I've seen you do before, to convert an option<result<..>> to the opposite?

Reply:
Yeah that works, but I'm not sure if it's easier to read. Anyway, added in [ea94c58](https://github.com/NomicFoundation/hardhat/pull/4452/commits/ea94c58d6e904bcbeb83f76dd1c4e2b9137040bf) in the hope we can resolve this 😅